packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageViewWrapper.kt

```diff
@@ -182,6 +184,18 @@ class ExpoImageViewWrapper(context: Context, appContext: AppContext) : ExpoView(
     activeView.setBorderColor(index, rgb, alpha)
   }
 
+  fun setIsAnimating(isAnimating: Boolean) {
+    val resource = activeView.drawable
+
+    if (resource is Animatable) {
+      if (isAnimating) {
+        resource.start()
```

Comment:
just a blind guess, but shouldn't the if-else be inverted?

Reply:
Ah, no.  Whatever the passed in value for `isAnimating` is is what we want to set. But looking at it I don't like the name `isAnimating` since it does imply that is the current value rather than the new value we are going to set. I'll change that.